Rubberduck.Core/UI/UnitTesting/TestExplorerControl.xaml

```diff
@@ -4,7 +4,7 @@
              xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
              xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
              xmlns:local="clr-namespace:Rubberduck.UI.UnitTesting"
-             xmlns:unitTesting="clr-namespace:Rubberduck.UnitTesting"
+             xmlns:vms="clr-namespace:Rubberduck.UI.UnitTesting.ViewModels"
```

Comment:
Might be just me, but I would have gone with `vm` - the `s` / pluralization doesn't seem necessary, given it's a namespace (and thus can be _assumed_ to contain multiple types)

Reply:
That said I'm not against `Rubberduck.UI.UnitTesting.ViewModels` being plural ;-)